docs/release-notes/0.2/release-0.2.md

```diff
@@ -0,0 +1,61 @@
+# ML.NET 0.2 Release Notes
+
+We would like to thank the community for the engagement so far and helping us shape ML.NET.
+
+Today we are releasing ML.NET 0.2. This release focuses on addressing questions/issues, adding clustering to the list of supported machine learning tasks, enabling using data from memory to train models, easier model validation, and more.
+
+### Installation
+
+Supported platforms: Windows, MacOS, Linux (see [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more detail)
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Release Notes
+
+Below are some of the highlights from this release.
+
+#### New machine learning tasks: clustering
+
```

Comment:
This formatting scheme does not correspond to the formatting used for 0.1 release notes. There was a `###` level 3 release notes header in those, but followed by a list style structure. I actually preferred that structure to this structure, where we are using, seemingly, level *four* headers to refer to individual points which, as near as I can tell, correspond to individual sentences.

Reply:
Changed to the previous format.